tests/check/check_h26xsigned_auth.c

```diff
@@ -1211,7 +1211,7 @@ START_TEST(recurrence)
   int recurrence = 3;
 
   nalu_list_t *list =
-      create_signed_nalus_recurrence("IPPIPPIPPIPPIPPIPPI", settings[_i], recurrence);
+      create_signed_nalus_recurrence("IPPIPPIPPIPPIPPIPPI", settings[_i], recurrence, false);
```

Comment:
In all places where `create_signed_nalus_recurrence()` is called `new_private_key = false`. I think we can set that implicitly and keep the API as before.